packages/app/utils/getChainName.js

```diff
@@ -0,0 +1,38 @@
+export const getChainName = (chainId) => {
```

Comment:
could we use the existing Network Name map?

https://github.com/CabinDAO/Passports/blob/main/packages/app/components/constants.ts#L13-L19